src/pages/home/report/withReportAndReportActionOrNotFound.tsx

```diff
@@ -37,12 +38,21 @@ type OnyxProps = {
     isLoadingReportData: OnyxEntry<boolean>;
 };
 
-type ComponentProps = OnyxProps &
-    WindowDimensionsProps & {
-        route: RouteProp<{params: {reportID: string; reportActionID: string}}>;
-    };
+type WithReportAndReportActionOrNotFound = OnyxProps &
```

Comment:
Why this change?

Reply:
Nvm this, now it's updated with the latest main and I adapted it to work with IOU